lib/game_of_life/universe/template.ex

```diff
@@ -1,97 +1,100 @@
+  alias GameOfLife.Universe.Dimensions
+  alias GameOfLife.Cell.Position
+
+  def dimensions(:beacon), do: %Dimensions{width: 6, height: 6}
+  def dimensions(:pulsar), do: %Dimensions{width: 17, height: 17}
+  def dimensions(:penta_decathlon), do: %Dimensions{width: 11, height: 18}
 
   def initial_state(:beacon) do
     [
-      {1, 1},
-      {1, 2},
-      {2, 1},
-      {3, 4},
-      {4, 3},
-      {4, 4}
+      %Position{x: 1, y: 1},
+      %Position{x: 1, y: 2},
+      %Position{x: 2, y: 1},
+      %Position{x: 3, y: 4},
+      %Position{x: 4, y: 3},
+      %Position{x: 4, y: 4}
     ]
   end
 
   def initial_state(:pulsar) do
     [
-      {4, 2},
-      {5, 2},
-      {6, 2},
-      {10, 2},
-      {11, 2},
-      {12, 2},
-      {2, 4},
-      {7, 4},
-      {9, 4},
-      {14, 4},
-      {2, 5},
-      {7, 5},
-      {9, 5},
-      {14, 5},
-      {2, 6},
-      {7, 6},
-      {9, 6},
-      {14, 6},
-      {4, 7},
-      {5, 7},
-      {6, 7},
-      {10, 7},
-      {11, 7},
-      {12, 7},
-      {4, 9},
-      {5, 9},
-      {6, 9},
-      {10, 9},
-      {11, 9},
-      {12, 9},
-      {2, 10},
-      {7, 10},
-      {9, 10},
-      {14, 10},
-      {2, 11},
-      {7, 11},
-      {9, 11},
-      {14, 11},
-      {2, 12},
-      {7, 12},
-      {9, 12},
-      {14, 12},
-      {4, 14},
-      {5, 14},
-      {6, 14},
-      {10, 14},
-      {11, 14},
-      {12, 14}
+      %Position{x: 4, y: 2},
+      %Position{x: 5, y: 2},
+      %Position{x: 6, y: 2},
+      %Position{x: 10, y: 2},
+      %Position{x: 11, y: 2},
+      %Position{x: 12, y: 2},
+      %Position{x: 2, y: 4},
+      %Position{x: 7, y: 4},
+      %Position{x: 9, y: 4},
+      %Position{x: 14, y: 4},
+      %Position{x: 2, y: 5},
+      %Position{x: 7, y: 5},
+      %Position{x: 9, y: 5},
+      %Position{x: 14, y: 5},
+      %Position{x: 2, y: 6},
+      %Position{x: 7, y: 6},
+      %Position{x: 9, y: 6},
+      %Position{x: 14, y: 6},
+      %Position{x: 4, y: 7},
+      %Position{x: 5, y: 7},
+      %Position{x: 6, y: 7},
+      %Position{x: 10, y: 7},
+      %Position{x: 11, y: 7},
+      %Position{x: 12, y: 7},
+      %Position{x: 4, y: 9},
+      %Position{x: 5, y: 9},
+      %Position{x: 6, y: 9},
+      %Position{x: 10, y: 9},
+      %Position{x: 11, y: 9},
+      %Position{x: 12, y: 9},
+      %Position{x: 2, y: 10},
+      %Position{x: 7, y: 10},
+      %Position{x: 9, y: 10},
+      %Position{x: 14, y: 10},
+      %Position{x: 2, y: 11},
+      %Position{x: 7, y: 11},
+      %Position{x: 9, y: 11},
+      %Position{x: 14, y: 11},
+      %Position{x: 2, y: 12},
+      %Position{x: 7, y: 12},
+      %Position{x: 9, y: 12},
+      %Position{x: 14, y: 12},
+      %Position{x: 4, y: 14},
+      %Position{x: 5, y: 14},
+      %Position{x: 6, y: 14},
+      %Position{x: 10, y: 14},
+      %Position{x: 11, y: 14},
+      %Position{x: 12, y: 14}
     ]
   end
 
   def initial_state(:penta_decathlon) do
     [
-      {4, 5},
```

Comment:
definitely like the switch to named arguments here, makes it much easier to read